search/filters/FulltextFilter.php

```diff
@@ -73,5 +73,18 @@ public function getDbName() {
 
 		return parent::getDbName();
 	}
+	
+	private function prepareColumns($columns) {
```

Comment:
If this is going to get into a patch release, it really shouldn't be anything other than private??